### ISSUE TITLE
builder.Dockerfile on Ubuntu 20.04

### ISSUE THREAD

**Issue by aeroned** [google/fuzzbench#1275]
I was trying to implement the latest version of AFLGo with the base-image given in the fuzzbench, however I am facing some build complications with the base-image's version of Ubuntu (xenial). Is it possible to create the builder.Dockerfile on Ubuntu 20.04?

**Comment by jonathanmetzman** [comment]
We will switch over soon.
I'm busy now with other things but probably in a week or two I can try to do this.

**Comment by aeroned** [comment]
Apologies for bothering. Any updates on the upgraded version?

**Comment by jonathanmetzman** [comment]
Sorry I didn't get any time to try this. Not sure when I will actually get a chance. I think there will be a lot of issues when we upgrade, particularly for the bug-based benchmarks, and I'm not sure I have the time to fix them right now.

**Comment by maxammann** [comment]
I did a quick experiment with just bumping xenial -> focal. It failed at using some apt repository, which should be fairly easy to fix.

What are the main concerns with updating to ubuntu 20? Do you think a lot of benchmarks would fail?

If that is the case, should we maybe just release a fuzzbench 2 and remove all failing benchmarks?

**Comment by jonathanmetzman** [comment]
> I did a quick experiment with just bumping xenial -> focal. It failed at using some apt repository, which should be fairly easy to fix.
> 
> What are the main concerns with updating to ubuntu 20? Do you think a lot of benchmarks would fail?

Yeah, there's about 50 benchmarks and 100 fuzzers so I've been dragging my feet on making sure all combinations work (this is somewhat overestimated as a lot of the fuzzers are near clones)

> 
> If that is the case, should we maybe just release a fuzzbench 2 and remove all failing benchmarks?

Yeah I pretty much plan on doing this.
